src/lib/features/feature-toggle/tests/feature-toggles.auth.e2e.test.ts

```diff
@@ -153,10 +158,14 @@ test('Should read flag creator', async () => {
         TEST_AUDIT_USER,
     );
 
-    await db.stores.featureToggleStore.create('default', {
-        name,
-        createdByUserId: user.id,
-    });
+    await app.services.featureToggleService.createFeatureToggle(
```

Comment:
had to switch to a service call so that an event is generated for the collaborators calculation